src/winetricks

```diff
@@ -6792,6 +6792,25 @@ load_dxvk65()
     helper_dxvk "$file1" 0 "3.10" "1.0.76"
 }
 
+w_metadata dxvk70 dlls \
+    title="Vulkan-based D3D10/D3D11 implementation for Linux / Wine (0.70)" \
+    publisher="Philip Rebohle" \
+    year="2018" \
+    media="download" \
+    file1="dxvk-0.70.tar.gz" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d10.dll" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d10_1.dll" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d10core.dll" \
+    installed_file1="$W_SYSTEM32_DLLS_WIN/d3d11.dll" \
+    installed_file2="$W_SYSTEM32_DLLS_WIN/dxgi.dll"
```

Comment:
Redefining the same variable 4 times doesn't make much sense.

Note that currently only installed_file1 is checked. See https://github.com/Winetricks/winetricks/issues/988